contextual_orchestrator/orchestrator.py

```
@@ -230,6 +238,7 @@ def __init__(
     @staticmethod
     def _build_ssl_context(ca_bundle: str | None, verify_tls: bool) -> ssl.SSLContext:
         if not verify_tls:
+            # nosemgrep: python.lang.security.unverified-ssl-context.unverified-ssl-context
             return ssl._create_unverified_context()  # nosec B323 - explicit dev-only provider TLS opt-out.
```

Ruff (0.16.1)

[error] Python allows using an insecure context via the `_create_unverified_context` that reverts to the previous behavior that does not validate certificates or perform hostname checks.

(S323)